src/civic/publication.js

```diff
@@ -0,0 +1,55 @@
+const _asco = require('../asco');
+const _pubmed = require('../asco');
```

Comment:
should this be `require('../pubmed')`

Reply:
yes! That must have got missed when I split this into a sep file. good catch!

Reply:
fixed!